pandas/tests/test_base.py

```diff
@@ -40,64 +39,6 @@
 import pandas.util.testing as tm
 
 
-class CheckStringMixin:
```

Comment:
were these just not running before?

Reply:
No, as the class name doesn't start with "Test", pytest is ignoring them. And as a mixin they are only used in `test_frozen`.

I validated that by comparing the number of tests being executed in `test_base.py` on master vs this branch